Jenkinsfile

```diff
@@ -8,8 +8,8 @@ pipeline {
   }
 
   tools {
-      maven 'mvn-3.5.2'
-      jdk 'jdk8'
+      maven 'mvn-3.9.9'
+      jdk 'jdk17'
```

Comment:
doesn't seem to be using any CI, and well these would need to be updated for this to keep working...